app/src/main/java/org/breezyweather/common/source/LocationPreset.kt

```diff
@@ -62,7 +62,8 @@ enum class LocationPreset(
     // Do NOT set up other sources as only 中国 source is not rate-limited by the Great Firewall
     CHINA("china"),
     HONG_KONG("openmeteo", alert = "wmosevereweather", normals = "accu"),
-    ISRAEL("ims", airQuality = "openmeteo", pollen = "openmeteo", minutely = "openmeteo");
+    ISRAEL("ims", airQuality = "openmeteo", pollen = "openmeteo", minutely = "openmeteo"),
+    TAIWAN("cwa");
```

Comment:
I will remove this after merging, as I cannot make users add a location and then find out they can't have fresh data because the source that is rate-limited; that's not a good experience.

I will still probably bundle an API key but it will only be usable at the beginning of the day until the daily limit is reached. Then, you have to wait for each daily reset. That's what is currently being done with OpenWeather.

Reply:
Oh! I don't know how to bundle an API key without making it visible in the source code. I'll leave that with you.

The default allowance for CWA's API key is 20,000 calls and 2GB of transfer per day.

One can apply for an upgrade to advanced membership. The upgrade is free/gratis, but requires filling in name and phone number, as well as manual approval by CWA staff. The allowance for the advanced API key is 300,000 calls and 20GB of transfer per day.

Reply:
You can't. GitHub is adding the keys when it builds the release. We also prefer to only use keys registered by the project in official releases ;)